packages/main/src/Popover.ts

```diff
@@ -849,3 +849,7 @@ Popover.define();
 export default Popover;
 
 export { instanceOfPopover };
+
+export type {
```

Comment:
No need to export type for the class   - `export default Popover;` contains both the class and the type.

You just need to use import type Popover in the NotificationList*